source/packages/services/events-processor/src/api/apigwtrigger/apigwtrigger.service.ts

```diff
@@ -0,0 +1,41 @@
+/*********************************************************************************************************************
```

Comment:
To be consistent, as this controller is dealing with messages, move it to source/packages/services/events-processor/src/api/messages/apigwtrigger.service.ts.

Reply:
Thanks! I fixed it according to your suggestion.